src/System.Security.Cryptography.Pkcs/tests/SignedCms/SignedCmsTests.cs

```diff
@@ -500,6 +500,87 @@ public static void AddFirstSigner_RSA(SubjectIdentifierType identifierType, bool
             cms.CheckSignature(true);
         }
 
+        [Fact]
+        public static void AddSigner_RSA_EphemeralKey()
+        {
+            using (RSA rsa = RSA.Create())
+            using (X509Certificate2 publicCertificate = Certificates.RSA2048SignatureOnly.GetCertificate())
+            using (X509Certificate2 certificateWithKey = Certificates.RSA2048SignatureOnly.TryGetCertificateWithPrivateKey(exportable: true))
+            {
+                if (certificateWithKey == null)
+                {
+                    return;
+                }
+                RSAParameters rsaParameters;
+                using (RSA privateKey = certificateWithKey.GetRSAPrivateKey(), exportableKey = privateKey.MakeExportable())
```

Comment:
Rather than use the multi-variable initializer, please split to two using statements  (applies to all of them)